src/collections/posts/getting-started.md

```diff
@@ -10,38 +10,48 @@ image: /static/uploads/posts/beat-saber-square.jpg
+Playing custom maps does not require mods, although using mods is generally the simplest method to download them. Alternatively, if you prefer not to install any mods, you can manually download the custom maps and place them in the appropriate folder within your game's directory. For this, you need to find the map files and copy them to a specific location within your game's files. However, the mod approach is still recommended.
 
 Regardless of your method, you will most likely be getting the maps from [BeatSaver](https://beatsaver.com), where you can search for and view various maps.
 
-# Modding
+# [Modding](/installing-the-mod-guide-necessary-for-any-custom-songs)
 
-Mods will allow you to download custom maps easily, compete in leaderboards, modify the game appearance, and many more. Some maps may require additional mods to be playable. For detailed modding instructions, refer to the [BSMG (Beat Saber Modding Group) Wiki](https://bsmg.wiki/beginners-guide.html) which includes a beginner's guide on how to mod the game.
+Mods will allow you to download custom maps easily, compete in leaderboards, modify the game's appearance, and much more. Some maps may require additional mods to be playable. For detailed modding instructions, refer to the [BSMG (Beat Saber Modding Group) Wiki](https://bsmg.wiki/beginners-guide.html) which includes a beginner's guide on how to mod the game.
 
 Note that certain maps may also require specific mods to play. These can include **Chroma**, **Noodle Extensions**, and **Mapping Extensions**. If you are interested in maps that require these mods, you can find them by using the "Requirements" filters on map selection pages.
 
-# Finding Good Maps
+# [Finding Good Maps](getting-started/custom-songs)
 
-Once you have broken out of beginner phase and regularly find yourself playing Expert+ maps you will find that the maps have quite a depth to them and distinct style to them.
+Once you have broken out of the beginner phase and regularly find yourself playing Expert+ maps you will find that the maps have quite a depth to them and distinct style to them.
 
-As BeastSaber, we want help you find great maps and such is the task of our curators.
+At BeastSaber, we want to help you find great maps and such is the task of our curators.
 
 Maps on BeatSaver can marked as curated which means one of the curators considered them fun. You can [filter maps on BeatSaver](https://beatsaver.com/?curated=true) to only show curated maps.
 
-Additionally, you can checkout the [maps of the week](/maps-of-the-week/1) which are recent maps the curator team has considered particularly noteworthy.
+Additionally, you can check out the [Maps of the Week](/maps-of-the-week/1) - which will almost always have the following characteristics:
 
-On our frontpage we also feature playlists, which are collections of maps that are all great and reviewed by the curators.
```

Comment:
I think this is duplicate content with About Curation? So one could consider linking that and hinting at more details there.